cm/job.py

```diff
@@ -729,20 +731,22 @@ def log_check(job_id, title, res, msg):
             err('JOB_NOT_FOUND', f'job #{job.id} has status "{job.status}", not "running"')
     except JobLog.DoesNotExist:
         err('JOB_NOT_FOUND', f'no job with id #{job_id}')
+    cl = CheckLog(job_id=job.id, title=title, message=msg, result=res)
+    cl.save()
+    return cl
```

Comment:
Is this `return cl` necessary here?

Reply:
It is not used now, but can be used in future. Just in case

Reply:
Ok.